src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -262,6 +251,14 @@ export class HuiCardPicker extends LitElement {
     this._loadCards();
   }
 
+  protected updated(changedProps) {
+    super.updated(changedProps);
+    const div = this.parentElement!.shadowRoot!.getElementById("content");
+    if (div) {
```

Comment:
Better add a check here that `this._filter` changed so it doesn't trigger on irrelevant updates